docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: '3.8'
+
+services:
+  web:
+    build: .
+    container_name: frontend
+    ports:
+      - "3000:80"
+    environment:
+      - NODE_ENV=production
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)